welcome_wizard/middleware.py

```diff
@@ -26,10 +26,7 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: d
         # If no user or they aren't authenticated, return to hit the default login redirect
         if not hasattr(request, "user") or not request.user.is_authenticated:
             return
-        if request.path == "/":
-            merlin_msg = f"<a href={reverse('plugins:welcome_wizard:dashboard')}>The Nautobot Welcome Wizard can help you get started with Nautobot!</a>"
-            messages.success(request, mark_safe(merlin_msg))  # nosec
-        elif request.path.endswith("/add/"):
+        if request.path.endswith("/add/"):
```

Comment:
Shouldn't all of the below also become part of the banner logic?

Reply:
In these, our goal was to use the messages feature to add the warning messages to the page, but I will try it with moving everything out of the middleware and just using Banners.